test/spec/function-references/br_table.txt

```diff
@@ -0,0 +1,78 @@
+;;; TOOL: run-interp-spec
+;;; STDIN_FILE: third_party/testsuite/proposals/function-references/br_table.wast
```

Comment:
Should the `update-spec-tests.py` script be updated to remove the exclusion of `function-references`?